trading_calendars/calendar_helpers.py

```diff
@@ -1,6 +1,9 @@
 import numpy as np
+import pandas as pd
 
 NANOSECONDS_PER_MINUTE = int(6e10)
+# TODO this is a negative number - can we just check that?
+NP_NAT = np.array([pd.NaT], dtype=np.int64)[0]
```

Comment:
A negative check seems reasonable to me, since the domain here is 24 hours. @llllllllll Thoughts on the call sites for this?